src/test/java/org/assertj/core/api/ComparatorFactory_doubleComparatorWithPrecision_Test.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2022 the original author or authors.
+ */
+package org.assertj.core.api;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.Comparator;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+
+class ComparatorFactory_doubleComparatorWithPrecision_Test {
+
+  private final ComparatorFactory INSTANCE = ComparatorFactory.INSTANCE;
+
+  @ParameterizedTest
+  @CsvSource({"0.111, 0.110, 0.001",
+    "0.111, 0.110, 0.001",
+    "0.12345, 0.12346, 0.00001",
+    "0.12346, 0.12345, 0.00001",
+    "0.7654321, 0.7654320, 0.0000001",
+    "0.7654320, 0.7654321, 0.0000001",
+    "1.2463, 1.2464, 0.0001",
+    "1.2464, 1.2463, 0.0001"})
+  void should_pass_for_expected_equal_to_actual_in_certain_range_max_min(Double expected, Double actual, Double precision) {
```

Comment:
the test name should not contain min max (same applies for other tests)

Reply:
Sorry, I did not see it. I have modified the test name.